crates/neon/src/types_impl/extract/try_into_js.rs

```diff
@@ -63,6 +65,30 @@ where
     }
 }
 
+impl<'cx, T, V> TryIntoJs<'cx> for Box<T>
+where
+    for<'a> &'a T: TryIntoJs<'cx, Value = V>,
+    V: Value,
+{
+    type Value = V;
+
+    fn try_into_js(self, cx: &mut Cx<'cx>) -> JsResult<'cx, Self::Value> {
+        self.as_ref().try_into_js(cx)
```

Comment:
This is wrong. For `Box` we can take ownership of the value, so we shouldn't be using a reference.